README.md

```diff
@@ -17,11 +17,18 @@
 
 ## Table of Contents
 
-- [Install](#install)
-- [Usage](#usage)
-- [API](#api)
-- [Contribute](#contribute)
-- [License](#license)
+- [datastore-pubsub](#datastore-pubsub)
```

Comment:
Did the TOC get generated? Looks like the main header formatting caused it to get added here

Reply:
Yeah, the Markdown All in One plugin for vscode does it.  Did you not want the header in the TOC?  It's added sections that were missing too.

Reply:
Feels silly to have `- [datastore-pubsub](#datastore-pubsub)` in there. The rest looks good.

Reply:
Yeah, I think we can remove it

Reply:
Fine, it's gone.